packages/roosterjs-editor-types/lib/interface/CustomReplacement.ts

```diff
@@ -21,7 +21,12 @@ export default interface CustomReplacement {
      * A callback to check if the string should be replaced
      * @param content the content where the string is
      * @param sourceString string to be replaced
+     * @param contentTag format of the string to be replaced
```

Comment:
I think you can change this comment to "contentTag  tagName of the html element that wraps contains the string"